lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -119,12 +119,7 @@ impl<NG: Deref<Target=NetworkGraph<L>>, L: Deref> RapidGossipSync<NG, L> where L
 		let default_htlc_minimum_msat: u64 = Readable::read(&mut read_cursor)?;
 		let default_fee_base_msat: u32 = Readable::read(&mut read_cursor)?;
 		let default_fee_proportional_millionths: u32 = Readable::read(&mut read_cursor)?;
-		let tentative_default_htlc_maximum_msat: u64 = Readable::read(&mut read_cursor)?;
-		let default_htlc_maximum_msat = if tentative_default_htlc_maximum_msat == u64::max_value() {
-			OptionalField::Absent
-		} else {
-			OptionalField::Present(tentative_default_htlc_maximum_msat)
-		};
+		let default_htlc_maximum_msat: u64 = Readable::read(&mut read_cursor)?;
```

Comment:
Seems bench and some test cases of `lightning-rapid-gossip-sync` are currently failing since the fallback value for `htlc_maximum_msat` is `u64::max_value()`, which our checks in `update_channel_intern` won't have. Should we default here to `MAX_VALUE_MSAT`, or just fix the input data to have sane `htlc_maximum_msat` fields? 

I'd tend towards the latter since going forward every update should have an `htlc_maximum_msat`?

@arik-so Do you have an opinion on that?

Reply:
@tnull told me that Rusty observed every single channel update setting its htlc maximum property, so I guess we can remove that special case handling from the rapid gossip sync protocol. Ought we increment the RGS protocol version for that?

Reply:
> Ought we increment the RGS protocol version for that?

As discussed offline, this will likely not be necessary since RGS likely has not a large userbase (if any) yet. I think in order to fix the failing test it would likely be easiest to create another graph snapshot in which all `ChannelUpdate`s have the `htlc_maximum_msat` field present?

Reply:
I'm still in discussions with Rusty about this, because I most certainly keep getting max-msat-less channel updates when syncing. If you guys have lightning nodes running, I encourage you to maybe create a print statement whenever you encounter one just as a sanity check for myself.

Reply:
Updated with new snapshot in https://github.com/lightningdevkit/rust-lightning/pull/1519/commits/c6ab8155de81092f6a5e8ac19f0c2be7c848f273, benchmark now passes.